external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/TopicFilter.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2017 The Apache Software Foundation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.storm.kafka.spout;
+
+import java.util.List;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.common.TopicPartition;
+
+public interface TopicFilter {
```

Comment:
must implement Serializable. Otherwise code didn't work for me. The trident [changes](https://github.com/hmcl/storm-apache/blob/35f2c0c71a28a065e64f523b861acd883b2f2101/external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/TopicFilter.java#L24) are already do this.